PhysicsTools/PatAlgos/interface/BaseMVAValueMapProducer.h

```diff
@@ -0,0 +1,322 @@
+#ifndef PhysicsTools_PatAlgos_BaseMVAValueMapProducer
```

Comment:
Any reason why you moved this file?

Reply:
The use of this class by PhysicsTools/NanoAOD (https://github.com/cms-sw/cmssw/blame/b32450010a0ed20cf00cfc9ba2e5626d329f1f5a/PhysicsTools/NanoAOD/plugins/BuildFile.xml)[https://github.com/cms-sw/cmssw/blame/b32450010a0ed20cf00cfc9ba2e5626d329f1f5a/PhysicsTools/NanoAOD/plugins/BuildFile.xml] is forbidden if the library is in another package's plugins directory, which is why this was also moved in https://github.com/cms-sw/cmssw/pull/31378/files

Reply:
Indeed, thanks :+1: